keras/utils/visualize_util.py

```diff
@@ -23,17 +23,29 @@ def model_to_dot(model, show_shapes=False, show_layer_names=True):
         model = model.model
     layers = model.layers
 
-    # first, populate the nodes of the graph
+    # Create graph nodes.
     for layer in layers:
         layer_id = str(id(layer))
+        
+        # Append a wrapped layer's label to node's label, if it exists
+        # (e.g. TimeDistributed(Dense()) becomes "TimeDistributedDense",
+        # Bidirectional(LSTM()) becomes "BidirectionalLSTM", and so on).
```

Comment:
I misread the code. Right. 

When it comes to layer names you are using a comma between the two layer names. That makes sense. Then for the layer class names you are concatenating the two names. I find that somewhat confusing. I would rather see something like:

```
Bidirectional(LSTM)
```
or 
```
Bidirectional[LSTM]
```
or at least
```
Bidirectional LSTM
```

To make it clear that the layer is a wrapper. Otherwise it looks like a single layer.